pkg/crdconversion/crdconversion.go

```diff
@@ -79,8 +80,14 @@ func (crdWh *crdConversionWebhook) run(stop <-chan struct{}) {
 	defer cancel()
 
 	webhookMux := http.NewServeMux()
-
-	// TODO (snchh): add handler and logic for conversion stratergy of each CRD in OSM
+	webhookMux.HandleFunc("/meshconfigconversion", serveMeshConfigConversion)
+	webhookMux.HandleFunc("/trafficaccessconversion", serveTrafficAccessConversion)
+	webhookMux.HandleFunc("/httproutegroupconversion", serveHTTPRouteGroupConversion)
+	webhookMux.HandleFunc("/multiclusterserviceconversion", serveMultiClusterServiceConversion)
+	webhookMux.HandleFunc("/egresspolicyconversion", serveEgressPolicyConversion)
+	webhookMux.HandleFunc("/trafficsplitconversion", serveTrafficSplitConversion)
+	webhookMux.HandleFunc("/tcproutesconversion", serveTCPRouteConversion)
+	webhookMux.HandleFunc("/ingressbackendspolicyconversion", serveIngressBackendsPolicyConversion)
```

Comment:
I meant to publish this comment earlier but missed it.

Since all these paths are for the resource conversion, could you change these to be of the form `/convert/<resource-name>`?

Reply:
updated